tools/lint-all

```diff
@@ -136,6 +139,9 @@ def custom_check_file(fn, rules, skip_rules=[]):
             except Exception:
                 print("Exception with %s at %s line %s" % (rule['pattern'], fn, i+1))
                 traceback.print_exc()
+    if lineFlag and '\n' not in lastLine:
+        print("No new line at the end of %s"%fn)
```

Comment:
In the Zulip style, this should be: `print("No new line at the end of %s" % (fn,))`